### PR TITLE
`2.3.2` has been released

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/airflow_bug_report.yml [hunk 1]
```diff
@@ -25,7 +25,8 @@ body:
         discuss Airflow 1.10, open a [discussion](https://github.com/apache/airflow/discussions) instead!
       multiple: false
       options:
-        - "2.3.1 (latest released)"
+        - "2.3.2 (latest released)"
+        - "2.3.1"
```
Comment (potiuk) [inline .github/ISSUE_TEMPLATE/airflow_bug_report.yml:29]: BTW. The list is getting longer and longer. I think we shoudl trim it down as it gets very difficult to reach "main" : )

Comment (potiuk) [reply .github/ISSUE_TEMPLATE/airflow_bug_report.yml:29]: Maybe we should use 2.0.* and 2.1.* instead of full list for those versions?

**Review by jedcunningham — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/airflow_bug_report.yml (hunk 1 above)
Comment (jedcunningham) [reply .github/ISSUE_TEMPLATE/airflow_bug_report.yml:29]: That works for me.

Comment (ephraimbuddy) [reply .github/ISSUE_TEMPLATE/airflow_bug_report.yml:29]: Works for me too. Should we do it here or in a separate PR?

Comment (potiuk) [reply .github/ISSUE_TEMPLATE/airflow_bug_report.yml:29]: Separate PR likely better :)

Comment (eladkal) [reply .github/ISSUE_TEMPLATE/airflow_bug_report.yml:29]: We can also move main to the top.

Comment (potiuk) [reply .github/ISSUE_TEMPLATE/airflow_bug_report.yml:29]: It's kinda good "main" is at the end (this template is mostly for users and "main" is really something for contributors)

**Pull request merged**